redis-persistence/src/main/java/com/netflix/conductor/dyno/RedisQueuesShardingStrategyProvider.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016 Netflix, Inc.
```

Comment:
Nit: Please use the latest copyright.

Reply:
Used the latest 2020 copyright header I found